nilearn/glm/first_level/design_matrix.py

```diff
@@ -281,7 +281,8 @@ def make_first_level_design_matrix(
         Particular attention should be given to the 'trial_type' key
         which defines the different conditions in the experimental paradigm.
 
-    hrf_model : {'glover', 'spm', 'spm + derivative', 'spm + derivative + dispersion',
+    hrf_model : {'glover', 'spm', 'spm + derivative',
+        'spm + derivative + dispersion',
         'glover + derivative', 'glover + derivative + dispersion',
         'fir', None}, optional
```

Comment:
It looks like this still doesn't render correctly:

![image](https://user-images.githubusercontent.com/8228902/129585602-2cf282a0-08a5-40d4-a95d-eed7dc67ebc8.png)